Plugins/TopSkyMaps.txt

```diff
@@ -6724,7 +6802,4 @@ SYMBOL:DOT:N069.59.24.855:E029.52.57.118
 SYMBOL:DOT:N070.00.24.805:E029.53.00.139
 SYMBOL:DOT:N070.01.24.756:E029.53.03.159
 SYMBOL:DOT:N070.02.24.706:E029.53.06.180
-SYMBOL:DOTB:N070.03.24.656:E029.53.09.201
-
-
-
+SYMBOL:DOT:N070.03.24.656:E029.53.09.201
```

Comment:
Okei?

Reply:
Sto DOTB istede for DOT :P

Reply:
![image](https://user-images.githubusercontent.com/59777657/184552716-5276dbd0-6826-4f24-b8aa-b884177c4b77.png)

Reply:
Det er meningen?

Reply:
Glem det, så feil. kommer en retting nå